### PR TITLE
Biospecimen Dashboard: Added checkin filter to return list of checked in particpants

### PR DESCRIPTION
**Pull request by abhinavjonnada82**
Title^^
This PR addresses following issue:
https://github.com/episphere/connect/issues/588

Related PR:
https://github.com/episphere/biospecimen/pull/545

### CODE REVIEW

File: utils/firestore.js [hunk 1]
```diff
@@ -791,6 +791,7 @@ const filterDB = async (queries, siteCode, isParent) => {
             if(key === 'connectId') query = query.where('Connect_ID', '==', parseInt(queries[key]));
             if(key === 'token') query = query.where('token', '==', queries[key]);
             if(key === 'studyId') query = query.where('state.studyId', '==', queries[key]);
+            if(key === 'checkedIn') query = query.where('331584571.266600170.135591601', '==', 353358909);
```
Comment (jhflorey) [inline utils/firestore.js:794]: Should we define conceptIDs in `fieldToConceptIdMapping.js`?

Comment (abhinavjonnada82) [reply utils/firestore.js:794]: not required since this is part of the query.

Comment (JoeArmani) [reply utils/firestore.js:794]: Select visit from dropdown menu -> baseline -> Check-in complete == yes

looks good

**Review by JoeArmani — APPROVED** [review]

**Review by we-ai — APPROVED** [review]

File: utils/firestore.js (hunk 1 above)
Comment (we-ai) [reply utils/firestore.js:794]: Variables for concept IDs will make the code more readable, for example:
```js
if (key === 'checkedIn') query = query.where(`{cid.dropdown}.{cid.baseline}.{cid.isCheckinComplete}`, '==', cid.yes);
```

**Pull request merged**